app.js

```diff
@@ -22,9 +21,13 @@ const opts = {
 
 persist(opts, (p) => {
   const app = choo()
-  app.use(log())
   app.use(p)
 
+  if (process.env.NODE_ENV === 'development') {
+    const log = require('choo-log')
```

Comment:
would it be bad to require this module anyway? would be cleaner to have all the requires at the top of the file